src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -82,6 +87,38 @@ public ReadOnlyAddressBook getAddressBook() {
         return favouritedList;
     }
 
+    /**
+     * Iterate through the list of Persons and return number of persons in
+     * the region given
+     * @param region is the region where buyers have their preference
+     *               for their potential property and sellers have their property in
+     */
+    @Override
+    public int getPersonsBasedOnRegion(String region) {
+        int totalPersons = 0;
+        if (Region.isValidRegion(region)) {
+            for (Person person : getFilteredPersonList()) {
```

Comment:
Why filteredPersonList here?

Reply:
filteredPersonList will return the total list of Person objects right?

Reply:
I think it would return the filtered list, so there would be problem if this command is called after a `find` command. 
Could refer to `ListCommand.execute()`.

Reply:
Alright I understand, thank you! I have made the necessary changes